snaproom-logging/configs/prometheus.yaml

```
@@ -0,0 +1,61 @@
+    environment: 'development'
+
+# Rule files for alerting
+rule_files:
+  - "/etc/prometheus/rules/*.yml"
+
+# Alerting configuration (if AlertManager is available)
+alerting:
+  alertmanagers:
+    - static_configs:
+        - targets:
+          - alertmanager:9093
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)